lib/yt/models/snippet.rb

```diff
@@ -29,6 +29,11 @@ def initialize(options = {})
       has_attribute :resource_id, default: {}
       has_attribute :thumbnails, default: {}
 
+      has_attribute :video_id
+      has_attribute :total_reply_count, default: 0
+      has_attribute :is_public, default: true
+      has_attribute :can_reply, default: true
```

Comment:
Look at https://github.com/Fullscreen/yt/blob/master/lib/yt/models/snippet.rb#L44-L47

We want to transform the response of YouTube into a (Ruby) boolean.